lib/Standalone/TransformUtils.cpp

```diff
@@ -86,16 +86,15 @@ getExpectedResultMemRefShape(SmallVector<OpFoldResult> sizes,
   return targetShape;
 }
 
-static Value getSliceOperand(OpBuilder &builder, linalg::LinalgOp linalgOp,
-                             Value operand,
-                             const SmallVector<OpFoldResult> &offsets,
-                             const SmallVector<OpFoldResult> &sizes,
-                             const SmallVector<OpFoldResult> &strides,
-                             unsigned desiredResultRank) {
+Value getSliceOperand(OpBuilder &builder, linalg::LinalgOp linalgOp,
```

Comment:
If this isn't static anymore, perhaps we should make it return `FailureOr` like the other one... Who know what will call this now...

Reply:
perhaps merge the two, then and force the caller to prepare the arguments every time

Reply:
Added a TODO. Will do later